src/python/bot/untrusted_runner/untrusted.py

```diff
@@ -137,8 +138,8 @@ def ListFiles(self, request, context):
     return file_impl.list_files(request, context)
 
   @wrap_servicer
-  def CopyFileTo(self, request, context):
-    return file_impl.copy_file_to_worker(request, context)
+  def CopyFileTo(self, request_iterator, context):
```

Comment:
I am not sure if this change is right way here. Can you double check. Does the protos need to be regenerated. This was done to fix this error:

| W:141, 2: Parameters differ from overridden 'CopyFileTo' method (arguments-differ)

Reply:
Also unsure why it didn't show up in local pylint run, but showed up on CI (both travis and googlecloud).